paper-server/patches/sources/net/minecraft/world/entity/boss/enderdragon/EnderDragon.java.patch

```diff
@@ -5,17 +5,20 @@
      private final int[] nodeAdjacency = new int[24];
      private final BinaryHeap openSet = new BinaryHeap();
 +    // Paper start
-+    private final net.minecraft.world.level.Explosion explosionSource; // Paper - reusable source for CraftTNTPrimed.getSource()
++    private final net.minecraft.world.level.ServerExplosion explosionSource; // Paper - reusable source for CraftTNTPrimed.getSource()
 +    @Nullable private BlockPos podium;
 +    // Paper end
  
      public EnderDragon(EntityType<? extends EnderDragon> entityType, Level level) {
          super(EntityType.ENDER_DRAGON, level);
-@@ -101,6 +_,7 @@
+@@ -101,6 +_,10 @@
          this.setHealth(this.getMaxHealth());
          this.noPhysics = true;
          this.phaseManager = new EnderDragonPhaseManager(this);
-+        this.explosionSource = new net.minecraft.world.level.ServerExplosion(level.getMinecraftWorld(), this, null, null, new Vec3(Double.NaN, Double.NaN, Double.NaN), Float.NaN, true, net.minecraft.world.level.Explosion.BlockInteraction.DESTROY); // Paper
++        // Paper start - initialize explosion source
++        this.explosionSource = new net.minecraft.world.level.ServerExplosion(level.getMinecraftWorld(), this, null, null, new Vec3(Double.NaN, Double.NaN, Double.NaN), Float.NaN, true, net.minecraft.world.level.Explosion.BlockInteraction.DESTROY);
++        this.explosionSource.yield = 0f;
```

Comment:
Based in how previously this value is directly send in the old method (that method not exists) then just set the yield to 0 for handle in events.